docs/quick-start.md

```diff
@@ -19,95 +13,17 @@ $ npm i --save @react-page/react-page
 
 That's it, congratulations! You should see something like this now:
 
 ![Example app](images/react-example-app.png)
 
 ## Readonly
 
-Simply provide a `readOnly={true}` prop to your editor from previous example. We will lazy load (using code splitting) the unnecessary UI if you use this approach. This can greatly decrease the size of this lib. Make sure to use this option if you don't require the editing capabilities of the editor and care about your app size (which you should!)
+Simply provide a `readOnly={true}` prop to your editor from previous example.  
+We will lazy load the unnecessary UI if you use this approach (using code splitting).
 
-## Advanced
+This can greatly decrease the size of this lib. Make sure to use this option if you don't require the editing capabilities of the editor and care about your app size (which you should!)
 
-The editor package from `import Editor from "@react-page/editor";` includes some pre-made UI. If you need to modify this UI, follow this approach instead:
-
-```jsx
-import React, { useState } from "react";
-
-// The editor core
-import Editor, { Editable, Provider, createEmptyState } from "@react-page/core";
-import "@react-page/core/lib/index.css"; // we also want to load the stylesheets
-// Require editor ui stylesheet
-import "@react-page/ui/lib/index.css";
-
-// Load some exemplary plugins:
-import slate from "@react-page/plugins-slate"; // The rich text area plugin
-import "@react-page/plugins-slate/lib/index.css"; // Stylesheets for the rich text area plugin
-import background from "@react-page/plugins-background"; // A plugin for background images
-import "@react-page/plugins-background/lib/index.css"; // Stylesheets for  background layout plugin
-
-// Define which plugins we want to use. We only have slate and background available, so load those.
-const plugins = {
-  content: [slate()], // Define plugins for content cells. To import multiple plugins, use [slate(), image, spacer, divider]
-  layout: [background({ defaultPlugin: slate() })] // Define plugins for layout cells
-};
-
-const editables = [createEmptyState()];
-
-const editor = new Editor({
-  plugins,
-  editables
-});
-
-const App = () => {
-  const [editorValue, setEditorValue] = useState(editables[0]);
-  return (
-    <Provider editor={oryEditor} dndBackend={finalBackend}>
-      {/*
-      The provider exposes all of editor's internal context. 
-      You can hook into this by connecting to it's redux state and altering and of it's state.
-      Please look into https://github.com/react-page/react-page/tree/master/packages/ui/src 
-      to learn about about how custom UI can be implemented.
-      */}
-      <Editable id={editorValue.id} onChange={setEditorValue} />
-    </Provider>
-  );
-};
-
-export default App;
-```
+[readonly.tsx](https://raw.githubusercontent.com/react-page/react-page-examples/4853e3c8bfe4d3e35d86a0e224b7aa7ce56e2b79/src/readonly.tsx ':include :type=code typescript')
```

Comment:
@macrozone the link needs to be changed once the pull request to the example repo is merged